nemo.yaml

```diff
@@ -0,0 +1,78 @@
+package:
+  name: nemo
+  version: 1.22.0
+  epoch: 0
+  description: conversational AI toolkit built for researchers working on ASR, TTS, LLMs, and NLP.
+  copyright:
+    - license: Apache-2.0
+  target-architecture:
+    - x86_64
+  options:
+    no-provides: true
+    no-depends: true
+  dependencies:
+    runtime:
+      - libgomp
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - ffmpeg-dev
+      - libsndfile-dev
+      - py3.11-installer
+      - py3.11-pip
+      - python-3.11
+      - python-3.11-dev
+      - samurai
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/NVIDIA/NeMo.git
+      tag: v${{package.version}}
+      expected-commit: 0b7467ebd158ef6876dd147189517b25ef626982
+
+  - runs: git submodule update --init --recursive
+
+  - runs: |
+      sed -i '/fasttext/d' requirements/requirements_nlp.txt
+      sed -i 's/megatron_core==0.4.0/megatron_core==0.3.0/' requirements/requirements_nlp.txt
+
```

Comment:
I know this is draft btw, so feel free to ignore this comment if not relevant, but looks like we're missing the build step?
 
i.e we're just setting up the virtualEnv and installing the dependencies, but not building nemo? Maybe this what your testing.

I guess that explains why there is no nemo package inside the virtualEnv (my comment below).

[See here](https://github.com/wolfi-dev/os/blob/9e21b9522815e253e6fe89102a999186d1e06c49/az.yaml#L26) for another example.